micro_sam/automatic_segmentation.py

```diff
@@ -230,7 +230,7 @@ def _convert_argval(value):
         model_type=args.model_type,
         checkpoint=args.checkpoint,
         device=args.device,
-        amg=args.amg,
+        amg=args.amg if args.amg else None,
```

Comment:
This can't really be corrected. We now have the following cases:
```
args.amg = None -> we pass on None
args.amg = False -> we pass on None
args.amg = False -> we pass on None 
```
so we never pass on `False`

Reply:
The condition here is:
- if `args.amg` is False, then `amg` is set to `None`, which makes an automatic choice depending on the model's decoder state availability.
- if `args.amg` is True, them `amg` forces to ignore the decoder.

(let me know if i misunderstood something)